list/LIST_HEADING_README.md

```diff
@@ -0,0 +1,163 @@
+|----------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| `CompoundStatement [@classCode]`                                                                                                             | fixed value = `CATEGORY`                                                                                                |
+| `CompoundStatement / id [@root]`                                                                                                             | new system generated UUID                                                                                               |
+| `CompoundStatement / code [@codeSystem='2.16.840.1.113883.2.1.3.2.4.15'] [@code='394841004'] [@displayName="Other category"] / originalText` | `resource.title` or not populated if not found                                                                          |
+| `CompoundStatement / statusCode [@code]`                                                                                                     | fixed value = `COMPLETE`                                                                                                |
+| `CompoundStatement / effectiveTime`                                                                                                          | from mapped [Encounter](../encounters/README.md) referenced in `list.encounter` <sup>1</sup><sup>2</sup><sup>3</sup>    |
+| `CompoundStatement / availabiltyTime [@value]`                                                                                               | `list.date` or from mapped [Encounter](../encounters/README.md) referenced in `list.encounter` <sup>4</sup><sup>5</sup> |
+| `CompoundStatement / components`                                                                                                             | contains one or more `CompoundStatements` mapped from 'list.entry' <sup>6</sup>                                         |
+
+1. When `Encounter` has `encounter.period.start` and `encounter.period.end` then values are set `effectiveTime / lowValue` & `effectiveTime / highValue` using `encounter.period.start` and `encounter.period.end` respectively
+2. When `Encounter` has `encounter.period.start` only, then `effectiveTime / center` is set by `encounter.period.start`
+3. If there is no `encounter.period` then value `effectiveTime / center [@nullFlavour="UNK"]` is used
+4. If `list.date` is not present and `encounter.period.start` is present then that value is used
+5. if `list.date` and `encounter.period.start` are not present `availabilityTime [@nullFlavour="UNK"]` is used
+6. each `list.entry` is mapped to `CompoundStatement` from the reference to the relevant clinical record
+
+<details><summary>Example XML</summary>
+
+```
+<component typeCode="COMP">
```

Comment:
Looking at the generation code I'd be expecting `contextConductionInd="true"` to be here 🤔 .

Reply:
added following discussion